kokoro/scripts/test/go_test.sh

```diff
@@ -53,6 +53,45 @@ track_flakiness
 # Avoids "fatal: detected dubious ownership in repository" errors on Kokoro containers.
 git config --global --add safe.directory "$(pwd)"
 
+# A helper for parsing YAML files.
+# Ex: VALUE=$(yaml ~/my_yaml_file.yaml "['a_key']")
+function yaml() {
+  python3 -c "import yaml;f=yaml.safe_load(open('$1'))$2;print(','.join(str(i) for i in f) if type(f)==list else f);"
+}
+
+function set_platforms() {
+  # if PLATFORMS is defined, do nothing
+  if [ -n "${PLATFORMS}" ]; then
+    return 0
+  fi
+  # if _LOUHI_TAG_NAME is defined, set TARGET and ARCH env vars by parsing it.
+  # Example value: louhi/2.46.0/shortref/windows/x86_64/start
+  if [ -n "${_LOUHI_TAG_NAME}" ]; then
+    local split_tag=(${_LOUHI_TAG_NAME//\// })
```

Comment:
This is kinda hard to read and weird (replace `/` by space, slam it into an array using `()`) and won't work if there are ever any spaces in these tags. What about:

```
TARGET="$(echo -n "${_LOUHI_TAG_NAME} | cut --delimiter="/" --fields=4)"
ARCH="$(echo -n "${_LOUHI_TAG_NAME} | cut --delimiter="/" --fields=5)"
```

adapted from https://stackoverflow.com/a/29903172.

Reply:
Done